tests/Rules/Function/IncludeFunction/IncludeFunctionRuleTest.twig

```diff
@@ -0,0 +1,16 @@
+{% include 'template.html' %}
+{% include ['template_a.html', 'template_b.html'] ignore missing with {'foo': 'bar'} only %}
+{% include 'template.html' only %}
```

Comment:
@smnandre This is fixed in `{{ include( 'template.html'  , false) }}`.

I think it's wrong, and should be `{{ include( 'template.html', [], false) }}`
Not sure if it's easy without named parameters...

Reply:
Solved in https://github.com/VincentLanglet/Twig-CS-Fixer/pull/178

Reply:
Damn really sorry to let so many mistakes :(